hw/ip/entropy_src/doc/registers.md

```diff
@@ -1,6 +1,3 @@
-# Registers
-
-<!-- BEGIN CMDGEN util/regtool.py -d ./hw/ip/entropy_src/data/entropy_src.hjson -->
```

Comment:
I think we've lost some lines here? (I'd expect to see the CMDGEN comment)

Reply:
Interesting, isn't the regtool supposed to write this? Or do we have to add this manually?

Reply:
The magic comment is supposed to mean "please replace everything that is after this line but before 'END CMDGEN' with the output of running the following command..."

This gets checked with `ci/scripts/check-cmdgen.py`. And you can put the correct text into the middle by using `util/cmdgen.py -u path/to/my/markdown/file`

... Oh!

I see that you've updated the PR to do that. Looks good to me :-)